nolearn/lasagne/visualize.py

```diff
@@ -338,29 +347,31 @@ def make_pydot_graph(layers, output_shape=True, verbose=False):
     return pydot_graph
 
 
-def draw_to_file(layers, filename, **kwargs):
+def draw_to_file(net, filename, **kwargs):
```

Comment:
May I ask why to do that? 

Let me make an argument. We pass the "net" for abstracting the implementation to the user and make the function more intuitive right? So why pass the layers?